### PR TITLE
Add support for PHP >= 7.3, PHPUnit 9, fix test errors

### PR DESCRIPTION
**Pull request by bebetoalves**

### CODE REVIEW

**Comment by bebetoalves** [comment]
@googlebot I signed it!

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,24 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (jfschmakeit) [inline phpunit.xml.dist:1]: Please reformat this file to follow the existing formatting style.

**Review by jfschmakeit — CHANGES_REQUESTED** [review]
Thanks for this pull request!

Could you please rebase it onto the latest changes and reformat your files? We just merged a pull request that addressed some of your changes to the `FiltersBuilderTest.php` file. 

Otherwise this looks good, including bumping the minimum PHP version to 7.3+ and 8.0+, plus updating the phpunit dependency.

**Comment by jfschmakeit** [comment]
@bebetoalves - We would like to merge this in and include this in the next release. It would be great if you could rebase your change and take a a look at the comment I added. Appreciate your help!

File: phpunit.xml.dist (hunk 1 above)
Comment (jfschmakeit) [reply phpunit.xml.dist:1]: We'll reformat this file in a follow up change.

**Comment by jfschmakeit** [comment]
Let's merge this pull request in and I will follow up on the formatting comment afterwards.

Thanks for submitting this pull request!

**Pull request merged**